uni/lib/main.dart

```diff
@@ -163,24 +170,33 @@ class MyApp extends StatefulWidget {
   State<MyApp> createState() => MyAppState();
 }
 
+final GlobalKey<NavigatorState> navigatorKey = GlobalKey<NavigatorState>();
```

Comment:
Can this be at least a static variable of the `MyApp` class? I don't like it being available under the global namespace very much